Resources/Textures/LobbyScreens/attributions.yml

```diff
@@ -0,0 +1,9 @@
+- files: ["supermatter.png", "robotics.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Veritius#2351 (257233913951289344) on discord / @Veritius on GitHub"
+  source: ""
+  
+- files: ["warden.png", "pharmacy.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Solbusaur#4343 on discord"
```

Comment:
Should probably include their actual GUID?

Reply:
I pinged them, no response. Where would I get their GUID?

Reply:
Pinged you instructions on discord